frontend/src/queries/schema.ts

```diff
@@ -678,6 +680,15 @@ export interface TrendsQuery extends InsightsQueryBase<TrendsQueryResponse> {
     breakdownFilter?: BreakdownFilter
 }
 
+export interface LogsQueryResponse extends AnalyticsQueryResponseBase<Record<string, any>[]> {}
+export type CachedLogsQueryResponse = LogsQueryResponse & CachedQueryResponseMixin
+
+export interface LogsQuery extends DataNode<LogsQueryResponse> {
+    kind: NodeKind.LogsQuery
+    /** Date range for the query */
+    dateRange?: DateRange
+}
```

Comment:
can we do an eventsquery but always filter for $log?

then we don't need any code or are there things forcing us?

Reply:
I guess this is nice since we could automagically grab console logs too 🤔